chapter06/dags/12_Inter_DAG_dependencies.py

```diff
@@ -0,0 +1,111 @@
+"""
+    Figure: 6.17
+"""
+
+
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+
+# ================================================ EXAMPLE 1 =================================================
+
+with DAG(
+    dag_id="figure_6_17_example_1_dag_1",
```

Comment:
Rename the dags to adhere to naming of the file convention?

Reply:
agree